packages/arb-token-bridge-ui/src/components/TransferPanel/ETHPrice.tsx

```diff
@@ -0,0 +1,43 @@
+import { useMemo } from 'react'
+
+import { useNativeCurrency } from '../../hooks/useNativeCurrency'
+import { useNetworks } from '../../hooks/useNetworks'
+import { useNetworksRelationship } from '../../hooks/useNetworksRelationship'
+import { useAppState } from '../../state'
+import { isNetwork } from '../../util/networks'
+import { useETHPrice } from '../../hooks/useETHPrice'
+import { formatUSD } from '../../util/NumberUtils'
+
+export function ETHPrice({
+  amountInEth,
+  showBrackets = false
+}: {
+  amountInEth: number
+  showBrackets?: boolean
+}) {
+  const {
+    app: { selectedToken }
+  } = useAppState()
+  const [networks] = useNetworks()
+  const { childChain, childChainProvider } = useNetworksRelationship(networks)
+  const nativeCurrency = useNativeCurrency({ provider: childChainProvider })
+  const isBridgingEth = selectedToken === null && !nativeCurrency.isCustom
+  const showPrice = useMemo(
+    () => isBridgingEth && !isNetwork(childChain.id).isTestnet,
+    [isBridgingEth, childChain.id]
+  )
```

Comment:
feels like all this logic doesn't really belong inside a component that's just supposed to show usd price for eth

i think it makes more sense for it to be outside, and do something like

```
{showPrice && <EthPrice />}
```

Reply:
This ETHPrice component is called in multiple places and moving the same set of logic into the parent component would be worse for performance